presto-main/src/main/java/com/facebook/presto/dispatcher/DispatchManager.java

```diff
@@ -89,6 +92,25 @@
 
     private final SecurityConfig securityConfig;
 
+    /**
+     * Instantiates a new Dispatch Manager.
```

Comment:
What does it do?

Reply:
rephrase and add a bit more color to the description